doc/generic/pgf/version-for-dvips/en/Makefile

```diff
@@ -9,13 +9,13 @@ pgfmanual.ps: $(doc).tex revisionfile
 	TEXINPUTS="$(TEXINPUTS)" $(latex) $(doc).tex
 	(egrep -q $(rerun) $(doc).log && TEXINPUTS="$(TEXINPUTS)" $(latex) $(doc).tex) || true
 	dvips -o $(doc).ps $(doc).dvi
-	gs -dNOPAUSE -sDEVICE=pdfwrite -dBATCH -dCompatibilityLevel=1.4 -sOutputFile=$(doc).pdf $(doc).ps
+	gs -dNOPAUSE -sDEVICE=pdfwrite -dBATCH -dCompatibilityLevel=1.4 -dALLOWPSTRANSPARENCY -sOutputFile=$(doc).pdf $(doc).ps
 
 test: $(doc)-test.tex revisionfile
 	TEXINPUTS="$(TEXINPUTS)" $(latex) $(doc)-test.tex
 	(egrep -q $(rerun) $(doc).log && TEXINPUTS="$(TEXINPUTS)" $(latex) $(doc)-test.tex) || true
 	dvips -o $(doc)-test.ps $(doc)-test.dvi
-	gs -dNOPAUSE -sDEVICE=pdfwrite -dBATCH -dCompatibilityLevel=1.4 -sOutputFile=$(doc)-test.pdf $(doc)-test.ps
+	gs -dNOPAUSE -sDEVICE=pdfwrite -dBATCH -dCompatibilityLevel=1.4 -dALLOWPSTRANSPARENCY -sOutputFile=$(doc)-test.pdf $(doc)-test.ps
```

Comment:
Is this new option required for users of the dvips driver now? If that is the case this should be clearly marked in the manual.

Reply:
Yes, if transparency is used, this new `gs` option must be added. I updated the documentation accordingly (in the drivers section below Installation and Configuration) and in the Transparency section of the TikZ manual.